vespalib/src/tests/array/array_test.cpp

```diff
@@ -333,26 +334,26 @@ struct UnreserveFixture {
     UnreserveFixture() : arr(1025, 7, alloc::Alloc::allocMMap(0))
     {
         EXPECT_EQUAL(1025u, arr.size());
-        EXPECT_EQUAL(2048u, arr.capacity());
+        EXPECT_EQUAL(round_up_to_page_size(1) == 64_Ki ? 16_Ki : 2_Ki, arr.capacity());
     }
 };
 
 TEST_F("require that try_unreserve() fails if wanted capacity >= current capacity", UnreserveFixture)
 {
-    EXPECT_FALSE(f.arr.try_unreserve(2048));
+    EXPECT_FALSE(f.arr.try_unreserve(32_Ki));
 }
 
 TEST_F("require that try_unreserve() fails if wanted capacity < current size", UnreserveFixture)
 {
-    EXPECT_FALSE(f.arr.try_unreserve(1024));
+    EXPECT_FALSE(f.arr.try_unreserve(1_Ki));
 }
 
 TEST_F("require that try_unreserve() succeedes if mmap can be shrinked", UnreserveFixture)
 {
     int *oldPtr = &f.arr[0];
     f.arr.resize(512);
     EXPECT_TRUE(f.arr.try_unreserve(1023));
-    EXPECT_EQUAL(1_Ki, f.arr.capacity());
+    EXPECT_EQUAL(round_up_to_page_size(1) == 64_Ki ? 16_Ki : 1_Ki, f.arr.capacity());
```

Comment:
Original test was close to the border, but is one lower to test that new capacity is larger than requested capacity. `page_ints() - 1` is probably a better argument to `try_unreserve()`, which results in the expected capacity being `page_ints()`